src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 20, 2023
+
+## AEP React Native libraries 5.0.0
+
+The following React Native libraries have been upgraded to use the Adobe iOS SDK 4.x. Starting with version 5.0.0 of the Adobe React Native SDK, all React Native libraries that share the same major version are compatible with each other.
```

Comment:
The following React Native libraries have been upgraded to use the version 4 of the Adobe Experience Platform Mobile SDK for iOS. 
Starting with this release, all React Native libraries that share the same major version are compatible with each other.